.drone.jsonnet

```diff
@@ -477,6 +477,8 @@ local integration_cilium = Step('e2e-cilium', target='e2e-qemu', privileged=true
   SHORT_INTEGRATION_TEST: 'yes',
   WITH_SKIP_BOOT_PHASE_FINISHED_CHECK: 'yes',
   WITH_CUSTOM_CNI: 'cilium',
+  WITH_KUBESPAN: 'true',
```

Comment:
I'm not sure if it'd be fair to run with KubeSpan always, as it might affect Cilium and we might miss a bug without KubeSpan?

Also should we enabled for `e2e-kubespan`? We could do another Cilium test with KubeSpan as the third entry?

Reply:
I'm pretty sure I tested it locally :crossed_fingers: